scratch/.gitignore

```
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

```suggestion
!.gitignore

```
just for sanity purposes, we might want to add a new line